packages/gobject-introspection/gobject-introspection.0.2/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Cédric Le Moigne <cedlemo@gmx.com>"
+authors: "Cédric Le Moigne <cedlemo@gmx.com>"
+homepage: "https://github.com/cedlemo/OCaml-GObject-Introspection"
+bug-reports: "https://github.com/cedlemo/OCaml-GObject-Introspection/issues"
+license: "GPL3"
+dev-repo: "git+https://github.com/cedlemo/OCaml-GObject-Introspection.git"
+synopsis: "OCaml bindings for the GObject-Introspection library (https://gi.readthedocs.io/en/latest/)"
+description: "This library provides bindings based on Ctypes for the libgirepository API (see https://developer.gnome.org/gi/stable/)"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.2"}
```
According to your dune-project file, you need at least dune 1.2, furthermore dune cannot be tagged as build dependency due to forward compatibility issues (see https://github.com/ocaml/opam-repository/issues/14185)